transcom/mymove
# INT I-13556 B-21376 part deaux

## [Issue ticket](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A1034466)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21376)

## Summary

Wanted a ToolTip. Here's yer ToolTip.

### How to test

1. Access MM as customer (need to set up `.envrc` to allow for UB shipments
2. Fill in orders info to qualify you for a UB shipment - submit
3. Go to your home page, you should now see the UB allowance & a tooltip at the top next to the weight allowance

## Screenshots
![Screenshot 2024-11-25 at 9 50 09 AM](https://github.com/user-attachments/assets/300f6059-269c-4bcb-88d0-db4e8aa4991e)
![Screenshot 2024-11-25 at 9 50 47 AM](https://github.com/user-attachments/assets/e9ec8a14-f014-44af-9a43-dac6b3fd7b1e)
